ted_sws/notice_publisher/services/notice_publisher.py

```diff
@@ -9,12 +9,13 @@
 from ted_sws.notice_publisher.adapters.sftp_publisher_abc import SFTPPublisherABC
 
 
-def publish_notice(notice: Notice, publisher: SFTPPublisherABC = SFTPPublisher(),
-                   remote_folder_path=config.SFTP_PATH) -> bool:
+def publish_notice(notice: Notice, publisher: SFTPPublisherABC = None,
+                   remote_folder_path: str = None) -> bool:
     """
         This function publishes the METS manifestation for a Notice in Cellar.
     """
-
+    publisher = publisher if publisher else SFTPPublisher()
```

Comment:
same as below:
use
a = a or b
Instead of
a = a if a else b

The same thing, but I think "or" version is more clear. Both are good